### PR TITLE
Adds a notification to load the the consignments modal from home

### PR DESCRIPTION
**Pull request by orta**
screen shot 2017-11-16 at 2 36 56 pm

### CODE REVIEW

**Comment by orta** [comment]
Fixes https://github.com/artsy/consignments/issues/91

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Missing Test Files:

- `src/lib/Components/Buttons/__tests__/DarkNavigationButton-tests.tsx`
- `src/lib/utils/__tests__/router-tests.ts`

If these files are supposed to not exist, please update your PR body to include "Skip New Tests".
  </td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: src/lib/Components/Consignments/Screens/Welcome.tsx [hunk 1]
```diff
@@ -10,7 +10,9 @@ import {
   ViewProperties,
 } from "react-native"
 
+import { Router } from "lib/utils/router"
```
Comment (mennenia) [inline src/lib/Components/Consignments/Screens/Welcome.tsx]: We have a router! 🎉

File: src/lib/Scenes/Home/index.tsx [hunk 2]
```diff
@@ -25,20 +28,31 @@ export const Tab: React.SFC<TabProps> = ({ children }) =>
...
+            <ForYouRenderer render={renderWithLoadProgress(ForYou)} />
+          </Tab>
+          <Tab tabLabel=" Auctions">
+            <SalesRenderer render={renderWithLoadProgress(Sales)} />
+          </Tab>
+        </ScrollableTabView>
+
+        <DarkNavigationButton
+          title="Sell works from your collection through Artsy"
+          onPress={this.openLink.bind(this)}
```
Comment (mennenia) [inline src/lib/Scenes/Home/index.tsx]: I thiiink binding wasn't necessary anymore as of a few months.

**Comment by mennenia** [comment]
This is exciting!

**Pull request merged**